src/js/data-model.jsx

```diff
@@ -1,82 +1,153 @@
-var React = require('react'),
-    _ = require('underscore');
+var React = require('react');
+var _ = require('underscore');
 
 
-var router = require('./router'),
-    facts = require('./facts');
+var CollapsibleText = require('./collapsible-text');
+var Dqa = require('./dqa');
+var Table = require('./table');
+var resources = require('./resources');
```

Comment:
how strongly do you feel about this? I have found this to be very inconvenient -- I keep forgetting to bump the semicolon down when adding a new line. I would prefer for each line to have its own var.